### DISCUSSION TITLE
Accounting for inflation

### DISCUSSION THREAD

**Discussion by dpatelisg** [USEPA/supply-chain-factors#24] [Q&A] [answered]
Is there an established methodology or best practice on applying inflation factors to adjust the emissions factors to match with the year of spend data?

**Comment by MoLi7** [comment, marked as answer]
Hi @dpatelisg 

The recommended practice to adjust EFs from year_A to year_B is:
`kg CO2e per year_A $  = kg CO2e per year_B $ * (year_B $ / year_A $)`
where `year_B $ / year_A $` is calculated using the inflation factor between `year_A` and `year_B`.

**Reply by arby1005** [reply]
Hi @MoLi7 
I wonder if I am applying the US EEIO emission factors to calculate emissions based on other currencies (i.e., other than USD), should I use the inflation rate of the USD or the currency that I am looking for?

**Reply by WesIngwersen** [reply]
You will have to convert to USD to use the factors. To do this, I would suggest using an international exchange rate to convert to USD where the exchange rate is an annual average for the year that matches the currency year of the factor that you are using, rather than using the [spot exchange rate](https://www.investopedia.com/terms/s/spotexchangerate.asp).

**Reply by WesIngwersen** [reply]
> Hi @dpatelisg
> 
> The recommended practice to adjust EFs from year_A to year_B is: `kg CO2e per year_A $ = kg CO2e per year_B $ * (year_B $ / year_A $)` where `year_B $ / year_A $` is calculated using the inflation factor between `year_A` and `year_B`.

Just to clarify, for the Supply Chain Factors we perform [this adjustment using our useeior R package function `adjustResultMatrixPrice` ](https://github.com/USEPA/supply-chain-factors/blob/660daff2a5dfbfab0b9259cad936ce1c2453095c/CalculateEmissionFactors.Rmd#L69) and we use commodity-specific inflation factors. We call these ρ ("rho" for the Greek letter we use to represent them) factors. See our USEEIO v2 [table of variables ](https://www.nature.com/articles/s41597-022-01293-7/tables/5) for definitions of variables including ρ

Thus, using an economy average 'inflation factor' will not result in the same price adjustment that we calculate.   In the current release of this repository for building v.1.1.1 factors, we can only adjust to USD 2018 but hope to update this to more recent years soon.

**Reply by rebecca-lcaresource** [reply]
Hi Wes, wondering if you would provide an example using actual data to walk through this using data from the https://catalog.data.gov/dataset/supply-chain-greenhouse-gas-emission-factors-v1-3-by-naics-6: The recommended practice to adjust EFs from year_A to year_B is: kg CO2e per year_A $ = kg CO2e per year_B $ * (year_B $ / year_A $) where year_B $ / year_A $ is calculated using the inflation factor between year_A and year_B.
example: $100 in 2022 is equivalent in purchasing power to about $109.27 today, an increase of $9.27 over 3 years. The dollar had an average inflation rate of 3.00% per year between 2022 and today, producing a cumulative price increase of 9.27%. To calculate 2025 CO2e using the 2022 emission factors, you then multiply the 2025 $ by 90.73% (100%-9.27%) and then by the 2022 emission factor (Supply Chain Emission Factors with Margins.) Or, do you multiply the 2022 emission factor by 90.73% and then the spend? As it's all multiplication - it doesn't matter but it would be consistent to adjust emission factors year on year. It's not perfect but it does account for inflation.
